README.md

```
@@ -1,2 +1,245 @@
+
+   Alternatively, you can start the application manually:
+
+   ```bash
+   mvn spring-boot:run
+   ```
+
+5. **Access the Application**
+
+   Once the application starts, access the backend at:
+
+   ```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)